src/presentation/transformers/reports/transformReportTestResultsToReport.test.ts

```diff
@@ -0,0 +1,31 @@
+import { aReportTestResult, aResult, aTestType } from '../../../test/domainFactories';
+import { transformReportTestResultsToReport } from './transformReportTestResultsToReport';
+import { ReportTestResult } from '../../../domain/model/reports/ReportTestResult';
+
+describe('transformReportTestResultsToReport', () => {
+  it('correctly transforms a ReportTestResult to Report data', () => {
```

Comment:
missing test for dynamic test result fields

Reply:
Good catch. 👍 Actually there was a logic bug in the test data set up here too.